os/arch/arm/src/amebasmart/amebasmart_serial.c

```diff
@@ -1315,10 +1490,9 @@ int up_lowgetc(void)
 {
 	uint8_t rxd;
 #ifdef CONFIG_UART4_SERIAL_CONSOLE
-	u32 IrqEn = LOGUART_GetIMR();
 	LOGUART_SetIMR(0);
 	rxd = LOGUART_GetChar(_FALSE);
-	LOGUART_SetIMR(IrqEn);
+	LOGUART_SetIMR(1);
```

Comment:
@lhenry-realtek  please revert this changes too, https://github.com/Samsung/TizenRT/pull/6514